### PR TITLE
feat(DEV-12231): custom Tabs for Receivables 🎲

### PR DESCRIPTION
**Pull request by radist2s**
- Added support for configuring custom tabs in Receivables Tables using MUI Theming.
- Maintained backward compatibility with default tabs: Invoices, Quotes, and Credit Notes.
- Updated the `<ReceivablesTable/>` component to use numeric tab indices instead of `ReceivablesTableTabEnum`.
- Changed default tab indices:
  - **Invoices**: 0
  - **Quotes**: 1
  - **Credit Notes**: 2

### CODE REVIEW

File: packages/sdk-react/src/components/receivables/ReceivablesTable/ReceivablesTable.stories.tsx [hunk 1]
```diff
@@ -16,7 +17,76 @@ export const FullPermissions: Story = {
...
+                  {
+                    label: 'Other Invoices',
+                    query: {
+                      type: 'invoice',
+                      sort: 'created_at',
+                      order: 'desc',
+                      status__in: [
+                        'accepted',
+                        'expired',
+                        'declined',
```
Comment (hkosova) [inline packages/sdk-react/src/components/receivables/ReceivablesTable/ReceivablesTable.stories.tsx:68]:
```suggestion
                        'issued',
                        'overdue',
```
`accepted`, `declined`, and `expired` are quote statuses.

File: packages/sdk-react/src/components/receivables/ReceivablesTable/ReceivablesTable.tsx [hunk 2]
```diff
@@ -32,127 +37,210 @@ export type ReceivablesTableProps = {
...
+ *           {
+ *             label: 'Other Invoices',
+ *             query: {
+ *               type: 'invoice',
+ *               sort: 'created_at',
+ *               order: 'desc',
+ *               status__in: [             // The "status" filter for the Tab will
+ *                 'accepted',             // only contain the values specified in "status__in"
+ *                 'expired',
+ *                 'declined',
```
Comment (hkosova) [inline packages/sdk-react/src/components/receivables/ReceivablesTable/ReceivablesTable.tsx:83]:
```suggestion
 *                 'issued',               // only contain the values specified in "status__in"
 *                 'overdue',
```

**Review by hkosova — CHANGES_REQUESTED** [review]

**Comment by radist2s** [comment]
@hkosova, could I ask you to validate the [status machine](https://github.com/team-monite/monite-sdk/blob/1c73f0fc9e81d3ab84143aec9a0df57a693afc6a/packages/sdk-react/src/components/receivables/ReceivableFilters/useReceivablesFilters.tsx#L57-L104)?

**Comment by hkosova** [comment]
> could I ask you to validate the [status machine](https://github.com/team-monite/monite-sdk/blob/1c73f0fc9e81d3ab84143aec9a0df57a693afc6a/packages/sdk-react/src/components/receivables/ReceivableFilters/useReceivablesFilters.tsx#L57-L104)?

@radist2s LGTM

**Pull request merged**